### PR TITLE
Update lightstep-tracer-python version to incoporate Lightstep-Access-Token header

### PR DESCRIPTION
**Pull request by cboppert**
This updates the tracer to version 4.0.4 and adds the changelog entry to reflect that this adds a header to thrift and http requests, Lightstep-Access-Token.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+CHANGELOG.md
+<a name="4.0.4"></a>
+## [4.0.2](https://github.com/lightstep/lightstep-tracer-python/compare/4.0.3...4.0.4)
```
Comment (carlosalberto) [inline CHANGELOG.md]: It should be "## [4.0.4]" (the previous release had also this wrong, sadly)

File: README.md [hunk 2]
```diff
@@ -89,12 +89,7 @@ as enabling use_thrift.
 
 ```python
 return lightstep.Tracer(
-    component_name='WhatIDoWhatIDidnt',
-    access_token='your-token-here',
-    collector_host='satellite.host',
-    collector_port=80,
-    collector_encryption='none',
-    verbosity=2,
+    ...
```
Comment (carlosalberto) [inline README.md]: I remember @austinlparker mentioning this one, but I feel we are mixing a few things here (simplifying the README with the access-token part). Lets go this way for now, though.

**Comment by austinlparker** [comment]
@cboppert could you resolve the merge conflicts here so we can get this merged in? thanks.

**Comment by cboppert** [comment]
@austinlparker @carlosalberto Hey, think I did that right, updated it to 4.1.1

**Comment by austinlparker** [comment]
Thanks!

**Pull request merged**